diff -Nurd kernel-source/scripts/Makefile kernel-source.mods/scripts/Makefile
--- kernel-source/scripts/Makefile	2017-06-26 15:06:19.669626507 -0500
+++ kernel-source.mods/scripts/Makefile	2017-06-26 15:21:13.000000000 -0500
@@ -10,6 +10,9 @@
 # check-lc_ctype: Used in Documentation/DocBook
 
 HOST_EXTRACFLAGS += -I$(srctree)/tools/include
+HOST_EXTRACFLAGS += -I$(STAGING_INCDIR_NATIVE)/
+HOST_EXTRACFLAGS += -I$(STAGING_INCDIR_NATIVE)/openssl
+HOST_EXTRACFLAGS += -L$(STAGING_LIBDIR_NATIVE)/
 
 hostprogs-$(CONFIG_KALLSYMS)     += kallsyms
 hostprogs-$(CONFIG_LOGO)         += pnmtologo

