Index: nokogiri-1.5.11/ext/nokogiri/extconf.rb
===================================================================
--- nokogiri-1.5.11.orig/ext/nokogiri/extconf.rb
+++ nokogiri-1.5.11/ext/nokogiri/extconf.rb
@@ -50,37 +50,16 @@ else
   lib_prefix = ''
 
   HEADER_DIRS = [
-    # First search /opt/local for macports
-    '/opt/local/include',
-
-    # Then search /usr/local for people that installed from source
-    '/usr/local/include',
-
     # Check the ruby install locations
     INCLUDEDIR,
-
-    # Finally fall back to /usr
-    '/usr/include',
-    '/usr/include/libxml2',
   ]
 
   LIB_DIRS = [
-    # First search /opt/local for macports
-    '/opt/local/lib',
-
-    # Then search /usr/local for people that installed from source
-    '/usr/local/lib',
-
     # Check the ruby install locations
     LIBDIR,
-
-    # Finally fall back to /usr
-    '/usr/lib',
   ]
 
   XML2_HEADER_DIRS = [
-    '/opt/local/include/libxml2',
-    '/usr/local/include/libxml2',
     File.join(INCLUDEDIR, "libxml2")
   ] + HEADER_DIRS
 
